sql/catalyst/src/main/scala/org/apache/spark/sql/util/ArrowUtils.scala

```diff
@@ -54,6 +54,8 @@ private[sql] object ArrowUtils {
         new ArrowType.Timestamp(TimeUnit.MICROSECOND, timeZoneId)
       }
     case NullType => ArrowType.Null.INSTANCE
+    case YearMonthIntervalType => Types.MinorType.INTERVALYEAR.getType
+    case DayTimeIntervalType => Types.MinorType.INTERVALDAY.getType
```

Comment:
Maybe the following code for consistency with the cases above:
```suggestion
    case YearMonthIntervalType => new ArrowType.Interval(IntervalUnit.YEAR_MONTH)
    case DayTimeIntervalType => new ArrowType.Interval(IntervalUnit.DAY_TIME)
```

Reply:
done